src/Scouting.elm

```diff
@@ -1,65 +1,121 @@
-module Scouting exposing (Msg, Model, init, update, main, view )
+module Scouting exposing (Model, Msg, init, main, update, view)
```

Comment:
`main` shouldn't be exposed - it's not really useful outside this module (and it can be easily created from `init`, `update` & `view` if it really is needed for some obscure reason).